model/fv_dynamics.F90

```diff
@@ -778,8 +782,8 @@ subroutine fv_dynamics(npx, npy, npz, nq_tot,  ng, bdt, consv_te, fill,
        call prt_mxm('liq_wat_dyn', q(isd,jsd,1,liq_wat), is, ie, js, je, ng, npz, 1.,gridstruct%area_64, domain)
        call prt_mxm('rainwat_dyn', q(isd,jsd,1,rainwat), is, ie, js, je, ng, npz, 1.,gridstruct%area_64, domain)
        call prt_mxm('ice_wat_dyn', q(isd,jsd,1,ice_wat), is, ie, js, je, ng, npz, 1.,gridstruct%area_64, domain)
-       call prt_mxm('snowwat_dyn', q(isd,jsd,1,snowwat), is, ie, js, je, ng, npz, 1.,gridstruct%area_64, domain)
-       call prt_mxm('graupel_dyn', q(isd,jsd,1,graupel), is, ie, js, je, ng, npz, 1.,gridstruct%area_64, domain)
+       !call prt_mxm('snowwat_dyn', q(isd,jsd,1,snowwat), is, ie, js, je, ng, npz, 1.,gridstruct%area_64, domain)
```

Comment:
Please use the same method as in lines 730 +/- - check if the index is larger than zero to print, otherwise skip. Don't comment out.  What about q_rimef?

Reply:
Good point. It reminds me that q_rimef is not  counted in Lagrangian_to_Eulerian step.

Reply:
BTW, we can not do similar thing here as in line 730, since the first argument of prt_mxm is an arbitrary character and we can not guarantee there is snowwat, graupel or q_rimef to print out in HWRF physics or GFDL MP